.buildkite/pipeline.yml

```
@@ -62,3 +62,15 @@ steps:
           - step: "int-test"
             allow_failure: true
     depends_on: "check"
+
+  - label: "Publish docker image"
+    key: "publish"
+    steps:
+      - label: "Build docker image"
+        command: ".buildkite/scripts/build_image.sh"
+        if: "!build.pull_request"
+        agents:
+          provider: "gcp"
```

Should this step be like this ? 

```suggestion
  - label: "Publish docker image"
    key: "publish"
    command: ".buildkite/scripts/build_image.sh"
    if: "!build.pull_request"
    agents:
      provider: "gcp"
```